### DISCUSSION TITLE
Consider OMOP support

### DISCUSSION THREAD

**Discussion by IntegratorBrad** [Project-MONAI/MONAI#3642] [Ideas]
OMOP is a standard that works with HL7 and other SDOs to support widescale multi-institutional representation of data. It may be a useful construct for representation of data to MONAI.

Details from their [website](https://www.ohdsi.org/data-standardization/the-common-data-model):

> The Observational Medical Outcomes Partnership (OMOP) CDM, now in its version 6.0, offers a solution unlike any other. OMOP found that disparate coding systems can be harmonized—with minimal information loss—to a standardized vocabulary.
> 
> Once a database has been converted to the OMOP CDM, evidence can be generated using standardized analytics tools. We at OHDSI are currently developing Open Source tools for data quality and characterization, medical product safety surveillance, comparative effectiveness, quality of care, and patient-level predictive modeling, but there are also other sources of such tools, some of them commercial. Read more about the OMOP CDM in the [Book of OHDSI](https://ohdsi.github.io/TheBookOfOhdsi/CommonDataModel.html).

**Comment by wyli** [comment]
Thanks Brad, I'm converting this into a discussion topic and will bring it up in the next dev meeting.
